lib/teracy-dev/location/git_synch.rb

```diff
@@ -15,12 +15,18 @@ def initialize
 
         current_git_version = `git --version`.gsub("git version", "").strip
 
-        required_git_version = '>= 2.20'
```

Comment:
I don't think we need this constraint as I suspect that:
- with git versions that support LANG env settings (>=2.19.1?), ENV['LANG'] setting below should get git to output in English.
- with git versions that do not support LANG env settings (< 2.19.1?), ENV['LANG'] does not take any effect and git will always output in English.

With that behavior, we can remove the required_git_version constraint.

Reply:
my current locale is Vietnamese and with git 2.16.0 => the output is always in English

```
macbook-pro:homebrew-core hoatle$ git --version
git version 2.16.0
macbook-pro:homebrew-core hoatle$ locale
LANG=
LC_COLLATE="C"
LC_CTYPE="UTF-8"
LC_MESSAGES="C"
LC_MONETARY="C"
LC_NUMERIC="C"
LC_TIME="C"
LC_ALL=
macbook-pro:homebrew-core hoatle$ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

  modified:   Formula/git.rb

no changes added to commit (use "git add" and/or "git commit -a")
macbook-pro:homebrew-core hoatle$ LANGUAGE= LANG=en_US.UTF-8 git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

  modified:   Formula/git.rb

no changes added to commit (use "git add" and/or "git commit -a")
macbook-pro:homebrew-core hoatle$ LANGUAGE= LANG=vi_VN.UTF-8 git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

  modified:   Formula/git.rb

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
my current locale is Vietnamese and with git 2.20.1 => the output is by default in Vietnamese, and LANG env var settings can be used to force in other languages (English in our case).

```
macbook-pro:homebrew-core hoatle$ git --version
git version 2.20.1
macbook-pro:homebrew-core hoatle$ git status
Trên nhánh master
Nhánh của bạn đã cập nhật với “origin/master”.

không có gì để chuyển giao, thư mục làm việc sạch sẽ
macbook-pro:homebrew-core hoatle$ LANGUAGE= LANG=en_US.UTF-8 git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean

```

Reply:
> with git versions that do not support LANG env settings (< 2.19.1?), ENV['LANG'] does not take any effect and git will always output in English.
sure it is always output in the language its located (not will always output in English), so we should suggest user to upgrade git version above 2.19.1 to take effect (will always output in English) right?

Reply:
I've rechecked by installing git 2.11.1 from sources with gettext (to enable locale) 

```
$ pwd
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core
$ git show de207300210813de108f7248490068faf253ca7d:Formula/git.rb > git-2.11.rb
```

And replace git.rb with git-2.11.rb to install git `$ brew install git --with-gettext`

after that, let's see:

```
$ git status
Trên nhánh master
Nhánh của bạn đã cập nhật với “origin/master”.
Các thay đổi chưa được đặt lên bệ phóng để chuyển giao:
  (dùng "git add <tập-tin>…" để cập nhật những gì sẽ chuyển giao)
  (dùng "git checkout -- <tập-tin>…" để loại bỏ các thay đổi trong thư mục làm việc)

	đã sửa:        Formula/git.rb

không có thay đổi nào được thêm vào để chuyển giao (dùng "git add" và/hoặc "git commit -a")
```


```
$ LANG=en_US.UTF-8 git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   Formula/git.rb

no changes added to commit (use "git add" and/or "git commit -a")
```

so I can conclude that LANG env var setting should work with many old git versions.

Reply:
@hoavt pls help me confirm if this works on your machine, thanks